src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -143,6 +143,7 @@ export default class FilingHistoryList extends Mixins(FilingMixin) {
   is (filing: ApiFilingIF): string {
     switch (true) {
       case filing.availableOnPaperOnly: return 'paper-filing' // must come first
+      case EnumUtilities.isTypeAgmExtension(filing): return 'agm-extension'
```

Comment:
This is where the specific filing template is assigned. Otherwise (eg, AGM Location Change), it uses the default filing template (line 158).